src/components/EventCard.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <EventCardSquare 
+    :item="item"
+    :width="width"
+    :height="height"
+    :host="host"
+  />
+</template>
+
+<script>
+import EventCardSquare from '@/components/EventCardSquare';
+
+export default {
+  name: 'event-card',
+
+  components: {
+    EventCardSquare
+  },
+
+  props: {
+    item: {
+      type: Object,
+    },
+    cellSize: {
+      type: Number,
+    },
+    gap: {
+      type: Number,
+    },
+    host: {
+      type: String,
+    }
+  },
+
+  data() {
+    return {
+    }
+  },
+
+  computed: {
+    width() {
+      const { cellSize, gap } = this;
+      const { is_highlighted } = this.item.event_listing;
```

Comment:
Generally I would convert `is_highlighted` to `isHighlighted` at some boundary, just to fit with js conventions.